### DISCUSSION TITLE
Sidebar navigation expanded to certain level

### DISCUSSION THREAD

**Discussion by jonaprieto** [squidfunk/mkdocs-material#7250] [Q&A]
I'm wondering if there is an option to expand the navigation menu to a certain depth level.

**Comment by kamilkrzyskow** [comment]
Hello @jonaprieto,
the `navigation.expand` feature expands the whole nav tree, there is no option to limit the depth by default means.
However, you can add some custom JavaScript to expand it yourself. 
- https://squidfunk.github.io/mkdocs-material/customization/#additional-javascript

I can provide some reference, as I've implemented a similar script on GMC, though it's not exactly the same case:
- https://github.com/Gothic-Modding-Community/gmc/blob/f6d9fecec5a125ebc7f719f80920962fc3618b5a/overrides/assets/javascripts/extra.js#L156-L203

**Reply by jonaprieto** [reply]
Oh thanks for the clarification and the reference. :)
